src/subdomains/supporting/fiat-payin/services/fiat-payin-sync.service.ts

```diff
@@ -57,7 +57,14 @@ export class FiatPayInSyncService {
 
     for (const refundedPayment of refundedPayments) {
       try {
-        await this.createCheckoutTx(refundedPayment);
+        const checkoutTx = await this.createCheckoutTx(refundedPayment);
+        if (checkoutTx)
+          await this.siftService.createChargeback({
+            $transaction_id: checkoutTx.transaction?.id.toString(),
+            $order_id: checkoutTx.transaction?.request?.id.toString(),
```

Comment:
You need to load the transaction and request in DB call. Every checkout TX has a transaction